compiler/src/main/scala/org/scalajs/core/compiler/Compat210Component.scala

```diff
@@ -27,6 +27,14 @@ trait Compat210Component {
     def originalName: Name = sys.error("infinite loop in Compat")
 
     def isLocalToBlock: Boolean = self.isLocal
+
+    def implClass: Symbol = NoSymbol
+
+    /** The real implementation from 2.12.0-M4 is `isTrait || isInterface`,
+     *  but we use this method in places where, previously, we had only
+     *  `isInterface`.
+     */
+    def isTraitOrInterface: Boolean = self.isInterface
```

Comment:
How hard is it to write an assertion that no one calls this from PrepJSInterop? IIUC, the method would not do the same in 2.11 and 2.12, there, right?